views/_footer.html

```diff
@@ -1,7 +1,12 @@
 {% block footer %}
-<footer class="site-footer">
-  <div class="footer-wrapper">
-    <p>Footer Text Goes Here</p>
+<footer class="footer">
+  <div class="footer--content">
+    <p>
```

Comment:
So, having this date, that's for essentially saying the copyright date? Otherwise, what is this date in there for?